force-app/main/default/classes/DMLOperations.cls

```diff
@@ -164,11 +186,18 @@ public with sharing class DMLOperations {
 		List<Account> accounts = [SELECT Id FROM Account WHERE Name = :accountName];
 
 		// if the Account exists, update the description
-
+        Account acc;
+        if (!accounts.isEmpty()){
+            acc = accounts[0];
+            acc.Description = 'Updated Account';
+        } else {
+            acc = new Account(Name = accountName, Description = 'New Account');
+        }
 		// otherwise, create a new Account and set the description
-
+       
 		// Upsert the Account
-		return null;
+        upsert acc;
+		return acc;
 	}
```

Comment:
Try to line up your brackets so it looks a little cleaner and for better readability